conpr/08_LE_Tasks/src/le/fj/Sum.java

```diff
@@ -1,23 +1,26 @@
 package le.fj;
 
 import java.util.Random;
-import java.util.concurrent.ForkJoinPool;
-import java.util.concurrent.RecursiveTask;
+import java.util.concurrent.Executors;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Callable;
 
 @SuppressWarnings("serial")
-public class Sum extends RecursiveTask<Integer> {
+public class Sum implements Callable<Integer> {
     final int THRESHOLD = 1000000;
     final int[] array;
     final int lo;
     final int hi;
+    final ExecutorService exec;
 
-    Sum(int[] array, int lo, int hi) {
+    Sum(int[] array, int lo, int hi, ExecutorService exec) {
         this.array = array;
         this.lo = lo;
         this.hi = hi;
+        this.exec = exec;
     }
```

Comment:
Adding an ExecutorService property to ensure that all subtasks are executed in the same one; see [here](https://github.com/ponder-lab/FHNW-Java-Projekte/pull/1/files#r1635483015) for more detail